dist/api.classic.js

```diff
@@ -111,7 +113,11 @@
             executeFn(disconnectFn)();
         }.bind(this), raiseError(errorFn, "connect error"));
     };
+    Device.prototype.isConnected = function() {
+        return this.connected;
+    };
     Device.prototype.disconnect = function(errorFn) {
+        this.connected = false;
```

Comment:
@thegecko Sounds like a good idea to let the adapter keep track of the connected state. Will look into this!